contrib/platform/test/com/sun/jna/platform/win32/PsapiTest.java

```diff
@@ -248,4 +248,31 @@ public void testGetPerformanceInfo() {
         assertTrue(Psapi.INSTANCE.GetPerformanceInfo(perfInfo, perfInfo.size()));
         assertTrue(perfInfo.ProcessCount.intValue() > 0);
     }
+
+    @Test
+    public void EnumProcesses() {
```

Comment:
At the risk of saying too much testing, I'm not sure we need both test classes you've written.  I think just moving the method you have in the util class here and not adding a new util test class should be ok.

Reply:
Oh, sorry, forgot that...

Reply:
I guess there are 2 sides to this coin.
While I agree that it is somewhat silly to test both, at the same time the checks are not completely redundant.
For example, the check for modulus 4 is an assumption taken in the java layer that will be hard to detect if the native code changes it's behavior. I don't say that this is likely, I just say that it would detect that scenario.
Then again, I used Advapi32Test / Advapi32UtilTest as a "template" on how to do the test and header part...